docs/header.html

```diff
@@ -7,12 +7,12 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
 <!--BEGIN PROJECT_NAME--><title>$title</title><!--END PROJECT_NAME-->
 <!--BEGIN !PROJECT_NAME--><title>$title</title><!--END !PROJECT_NAME-->
-<script type="text/javascript" src="jquery.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
```

Comment:
As long as it doesn't break any of the demos or `janus.js` functionality, why not... I only specified that as that's the version we're currently shipping and the one I used myself.